spec/rubocop/cop/rails/pluralization_grammar_spec.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::PluralizationGrammar do
+  subject(:cop) { described_class.new }
+  before(:each) do
+    inspect_source(cop, source)
+  end
+
+  shared_examples 'enforces pluralization grammar' do |method_name|
+    context "When #{method_name} is called on an unknown variable" do
+      context "when using the plural form ##{method_name}s" do
+        let(:source) { "some_variable.#{method_name}s" }
+        it 'does not register an offense' do
+          expect(cop.offenses).to be_empty
+        end
+      end
+
+      context "when using the singular form ##{method_name}" do
+        let(:source) { "some_method.#{method_name}" }
+
+        it 'registers an offense' do
+          expect(cop.offenses).to be_empty
```

Comment:
ah good catch. Indeed, at first, that's how I was implementing it, but it felt a bit complex, and then I started asking myself "well, what if the variable is called `one` and then RuboCop complains that the code says `one.day.ago` that wouldn't be great". I'll update the spec description for sure, but I'm inclined to leave the behavior to only lint for cases that are clearly wrong. How strongly do you feel about that suggestion?